java/src/jmri/implementation/DefaultSignalMastLogic.java

```diff
@@ -67,7 +68,7 @@ public DefaultSignalMastLogic(@Nonnull SignalMast source) {
                 source.setAspect(stopAspect);
             }
         } catch (Exception ex) {
-            log.error("Error while creating Signal Logic " + ex.toString());
+            log.error("Error while creating Signal Logic ", ex);
```

Comment:
The error produced after this change will include a stack trace while the one produced before does not.  Add the ’{}’ placeholder  at the end of the error message.

Reply:
Corrected, thanks